Makefile

```diff
@@ -67,10 +67,16 @@ bin/kind: test/k8s/Dockerfile
 test/k8s/_out/img: test/k8s/Dockerfile Cargo.toml Cargo.lock $(shell find . -type f -name '*.rs')
 	mkdir -p $(@D) && $(DOCKER_BUILD) -f test/k8s/Dockerfile --iidfile=$(@) --load  .
 
+.PHONY: test/py-flask
+test/py-flask:
+	$(DOCKER_BUILD) -t py-flask-app:latest -f $@/Dockerfile --load $@
```

Comment:
**suggestion:** Do we need a custom image? can't we use something like the `nginx` or `hello-world` images?

Reply:
That's a good suggestion!

Reply:
Done.